src/App/Pages/Vault/ViewPageViewModel.cs

```diff
@@ -284,9 +282,20 @@ public async Task<bool> LoadAsync(Action finishedLoadingAction = null)
             return true;
         }
 
-        public void CleanUp()
+        private async void StartCiphersTotpTick()
```

Comment:
This should be a `Task`. If `async void` is used then you need to try..catch the content given that the exception inside won't bubble up on `await` call and will just crash the app because of the Context